src/components/measureList/MeasureList.tsx

```diff
@@ -78,6 +89,28 @@ export default function MeasureList(props: {
   const versioningFeature = useFeature("measureVersioning");
   const targetMeasure = useRef<Measure>();
 
+  const [createVersionDialog, setCreateVersionDialog] = useState({
+    open: false,
+    measureId: "",
+  });
+  const handleDialogClose = () => {
+    setCreateVersionDialog({
+      open: false,
+      measureId: "",
+    });
+  };
+  const [snackBar, setSnackBar] = useState({
+    message: "",
+    open: false,
+    severity: null,
+  });
+  const handleSnackBarClose = (
+    event?: React.SyntheticEvent | Event,
+    reason?: string
```

Comment:
Are these props necessary ?

Reply:
@RohitKandimalla Thanks Rohit, they are not needed, so I removed them.